### ISSUE TITLE
Find: Cannot read property 'name' of undefined.

### ISSUE THREAD

**Issue by dexhorthy** [andjosh/clubhouse-cli#25]
Hi @andjosh -- looks like a cool project -- thanks! Was playing around and go this error running `club find`

``` $club find
Finding... ⢿ Error fetching stories: TypeError: Cannot read property 'name' of undefined
    at stories.map.map.filter.s (/home/dex/.nvm/versions/node/v8.7.0/lib/node_modules/clubhouse-cli/src/lib/stories.js:61:51)
    at Array.filter (<anonymous>)
    at /home/dex/.nvm/versions/node/v8.7.0/lib/node_modules/clubhouse-cli/src/lib/stories.js:52:8
    at Array.map (<anonymous>)
    at Object.listStories (/home/dex/.nvm/versions/node/v8.7.0/lib/node_modules/clubhouse-cli/src/lib/stories.js:21:20)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by andjosh** [comment]
@dexhorthy Thanks for the report! The filters were assuming that every story had an associated state. I was under the impression that was a requirement in Clubhouse's system, but that must not be the case.

It has been fixed and published in v0.10.4

**Comment by dexhorthy** [comment]
Sweet -- thanks for taking a look! Will check it out today
